python/src/aiconfig/registry.py

```diff
@@ -93,7 +94,7 @@ def clear_registry():
         ModelParserRegistry._parsers.clear()
 
     @staticmethod
-    def parser_ids() -> list:
+    def parser_ids() -> list[str]:
```

Comment:
This was typed as list unknown before, so update to list to prevent error where I need to use it above